netbox/project-static/css/base.css

```diff
@@ -62,8 +62,20 @@ footer p {
     }
 }
 
+/* Scroll the drop-down menus at or above 768px wide to match bootstrap's behavior for hiding dropdown menus */
+@media (min-width: 768px) {
+    .navbar-nav>li>ul {
+        max-height: 80vh;
+        overflow-y: scroll;
```

Comment:
Perhaps change this to auto instead

Reply:
Thanks for pointing it out. I'm on a mac so the scroll bar is only shown when scrolling.